src/ChatGPTBrowserClient.js

```diff
@@ -178,6 +178,71 @@ export default class ChatGPTBrowserClient {
         return response;
     }
 
+    async deleteConversation(conversationID) {
+        const url = this.options.reverseProxyUrl || 'https://chat.openai.com/backend-api/conversation';
+        const opts = {
+            method: 'PATCH',
+            headers: {
+                'Content-Type': 'application/json',
+                Authorization: `Bearer ${this.accessToken}`,
+                'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36',
+                Cookie: this.cookies || undefined,
+            },
+
+            body: JSON.stringify({
+                is_visible: false,
+            }),
+        };
+        const { debug } = this.options;
+
+        // eslint-disable-next-line no-async-promise-executor
+        const response = await new Promise(async (resolve, reject) => {
+            try {
+                let done = false;
+                await fetchEventSource(`${url}/${conversationID}`, {
```

Comment:
This should just use `fetch` as it's not an event stream response. I copied an example network request from my devtools:

```
await fetch("${url}/${conversationID}", {
    "credentials": "include",
    "headers": {
        "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/113.0",
        "Accept": "*/*",
        "Accept-Language": "en-US,en;q=0.5",
        "Content-Type": "application/json",
        "Authorization": "Bearer <TOKEN>",
        "Sec-Fetch-Dest": "empty",
        "Sec-Fetch-Mode": "cors",
        "Sec-Fetch-Site": "same-origin",
        "Pragma": "no-cache",
        "Cache-Control": "no-cache"
    },
    "referrer": "https://chat.openai.com/?model=text-davinci-002-render-sha",
    "body": "{\"is_visible\":false}",
    "method": "PATCH"
});
```